templates/rustdoc/header.html

```diff
@@ -96,6 +96,10 @@
                                             <a href="https://crates.io/crates/{{ krate.name }}" class="pure-menu-link" title="See {{ krate.name }} in crates.io">
                                                 {{ "cube" | fas(fw=true) }} Crates.io
                                             </a>
+
+                                            <a href="{{ crate_url | safe }}" class="pure-menu-link" title="See {{ krate.name }} in docs.rs">
+                                                {{ "cube" | fas(fw=true) }} Crate page on docs.rs
```

Comment:
Hmm, I don't think this is a great message because we're already on docs.rs. Maybe something like this?
```suggestion
                                                {{ "cube" | fas(fw=true) }} More information
```